sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -131,18 +147,65 @@ protected void beforeTest() {
     protected void afterTest() {
     }
 
+    /**
+     * Get test arguments need to run for the test framework based on the service version.
+     *
+     * @param serviceVersionList The service version argument for the parameterized tests.
+     * @param serviceVersionEnv Indicates whether the service version parameters need to rolling.
+     * @return Stream of arguments for parameterized test framework.
+     */
+    public static Stream<Arguments> getArgumentsFromServiceVersion(List<ServiceVersion> serviceVersionList,
+        String serviceVersionEnv) {
+        if (!ARGUMENTS_LIST.isEmpty()) {
+            return ARGUMENTS_LIST.stream();
+        }
```

Comment:
This should not be done as the incoming serviceVersionList can change for each call to this method.

Reply:
Revert the change.